spec/models/package_manager/packagist_spec.rb

```diff
@@ -1,83 +1,86 @@
     end
   end
 
   context "with an unmapped package" do
-   subject do
+    subject do
       {
         "name" =>	"librariesio/fakepkg",
         "description" => "A Libraries package.",
         "time" =>	"2012-09-18T06:46:25+00:00",
         "maintainers" => [],
         "versions" => {
-          "dev-master" => {"version" => "dev-master", "time" => "2020-01-08T08:45:45+00:00", "license" => ["BSD-3-Clause"], "name" =>	"librariesio/fakepkg", "description" => "A Libraries package."},
-          "1.2.3" => {"version" => "1.2.3", "time" => "2020-01-08T08:45:45+00:00", "license" => ["BSD-3-Clause"], "name" =>	"librariesio/fakepkg", "description" => "A Libraries package."},
-          "1.2.x-dev" => {"version" => "1.2.x-dev", "time" => "2020-01-08T08:45:45+00:00", "license" => ["BSD-3-Clause"], "name" =>	"librariesio/fakepkg", "description" => "A Libraries package."},
+          "dev-master" => { "version" => "dev-master", "time" => "2020-01-08T08:45:45+00:00", "license" => ["BSD-3-Clause"], "name" =>	"librariesio/fakepkg", "description" => "A Libraries package." },
+          "1.2.3" => { "version" => "1.2.3", "time" => "2020-01-08T08:45:45+00:00", "license" => ["BSD-3-Clause"], "name" =>	"librariesio/fakepkg", "description" => "A Libraries package." },
+          "1.2.x-dev" => { "version" => "1.2.x-dev", "time" => "2020-01-08T08:45:45+00:00", "license" => ["BSD-3-Clause"], "name" =>	"librariesio/fakepkg", "description" => "A Libraries package." },
         },
         "type" =>	"library",
-        "repository" => "https://github.com/librariesio/fakepkg"
+        "repository" => "https://github.com/librariesio/fakepkg",
       }
     end
 
     describe ".mapping" do
       it "maps correctly" do
         expect(described_class.mapping(subject)).to eq({
-          name: "librariesio/fakepkg",
-          description: "A Libraries package.",
-          homepage: nil,
-          keywords_array: [],
-          licenses: "BSD-3-Clause",
-          repository_url: "https://github.com/librariesio/fakepkg",
-        })
+                                                         name: "librariesio/fakepkg",
+                                                         description: "A Libraries package.",
+                                                         homepage: nil,
+                                                         keywords_array: [],
+                                                         licenses: "BSD-3-Clause",
+                                                         repository_url: "https://github.com/librariesio/fakepkg",
+                                                       })
       end
     end
 
     describe ".versions" do
       it "rejects dev branches that aren't really releases" do
-        expect(described_class.versions(subject, "librariesio/fakefpkg")).to eq([{number: "1.2.3", published_at: "2020-01-08T08:45:45+00:00"}])
+        VCR.use_cassette("packagist/testpkg") do
```

Comment:
This spec and the cassette are the only testing change, the rest is autolinting.